.env

```diff
@@ -17,6 +17,11 @@ REACT_APP_DATA_HASH_LOCAL=0xA5DAE7A114F1BD6BB9B3FF976150380A95CB18856212DB555C25
 # INFURA KEY
 REACT_APP_INFURA_KEY=586e7e6b7c7e437aa41f5da496a749f5
 
+# Cypress Integration Tests
+INTEGRATION_TEST_PRIVATE_KEY=
```

Comment:
Does it work?
I thought it should be in `cypress.env.json`

Reply:
https://docs.cypress.io/guides/guides/environment-variables#Option-2-cypressenvjson

Reply:
It worked locally, and tests passed in the branch.
Locally works because we use dotenv to override their envs.

In the e2e works because we have them here
https://github.com/cowprotocol/cowswap/blob/997afb6ab5a9fe02185e46a26b000556b606be71/.github/workflows/ci.yml#L230

It's true it shouldn't be there. I left them cause if someone duplicate the file to create the .env.local, then they will be able to set it up. I could create a local env file for those, but I don't see much benefit for introducing one more file. 

Happy to add it if you feel is better